cmd/spire-server/cli/agent/agent_test.go

```diff
@@ -51,6 +59,67 @@ func (s *agentTest) afterTest(t *testing.T) {
 	t.Logf("STDERR:\n%s", s.stderr.String())
 }
 
+func TestBanHelp(t *testing.T) {
+	test := setupTest(t, agent.NewBanCommandWithEnv)
+
+	test.client.Help()
+	require.Equal(t, `Usage of agent ban:
+  -registrationUDSPath string
```

Comment:
We don't need to support this flag, since this command is being introduced now and there are no compatibility concerns.

Reply:
it is inherited by `https://github.com/spiffe/spire/blob/main/cmd/spire-server/util/util.go#L133` that is common to all our cmds, an option is to reimplement it without that dependency?

Reply:
In that case I think that it's ok to leave it as is.